interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: 'this' is a phrase that allows a user to call values inside a function
 
-  Researched answer:
+  Researched answer: 'this' is a specific word that pertains to an object. It is used in methods to refer to data in an object.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: react is a program that is used to create an app
 
-  Researched answer:
+  Researched answer: React allows programmers to create a app and have it update the page without having to reload the page
```

Comment:
This feature is not only helpful to programmers, but as a user it is very nice to not have my entire page reload if something new is added to my feed on facebook.  The virtual DOM allows such a great developer and user experience!